### PR TITLE
Add Spanish translation

### PR DESCRIPTION
**Pull request by AlbertoFuente**
I'm translating documents into Spanish, I still have to translate the file readme.md, I hope you look good.

Cheers

### CODE REVIEW

**Comment by sindresorhus** [comment]
Awesome! Thanks for doing that @AlbertoFuente :)

You don't need to translate `maintaining.md`.

Do you know anyone that could review your translation?

**Comment by AlbertoFuente** [comment]
You are welcome @sindresorhus :). I can contact with someone to review the translation, if you know someone who can also do it would be nice. I thought I had to translate `maintaining.md`, anyway it's already translated, if that's okay I maintain, if not, tell me and I delete it. I think that the more documentation be translated will be better for everyone.

**Comment by sindresorhus** [comment]
> I think that the more documentation be translated will be better for everyone.

Sure, but there's really no point in translating that file as it's only for the AVA team which is expected to know English. It's just wasted effort.

> I can contact with someone to review the translation, if you know someone who can also do it would be nice.

That would be great. I don't know anyone that speaks Spanish.

**Comment by sindresorhus** [comment]
@forresst Mind reviewing that the format is correct? (Not the Spanish of course).

**Comment by sindresorhus** [comment]
@AlbertoFuente Let me know when it's done and I'll do a tweet about getting some spanish reviewers ;)

**Comment by AlbertoFuente** [comment]
Ok, I'll tell you when I have finished, I will talk with a friend to review the translation, I hope to finish tomorrow or Tuesday as much, by the way Happy New Year and great job with AVA. :+1:

**Comment by sindresorhus** [comment]
Happy new year 🎆

File: es-ES/contributing.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Contribuir a AVA
```
Comment (forresst) [inline es-ES/contributing.md]: As the [French version](https://github.com/sindresorhus/ava-docs/edit/master/fr_FR/contributing.md), add a header for the link to the original and to have a link on the differences, for example markdown formatted (Note, I do not speak Spanish, it is a translation from google translate) :

``` markdown
___
**Nota del traductor**

Esta es la traducción del archivo [contributing.md](https://github.com/sindresorhus/ava/blob/master/contributing.md). Aquí hay un [enlace](https://github.com/sindresorhus/ava/compare/65ae07c76b2a5927e6bcc00630a691c98f6e7c46...master#diff-cc4aac3e9be04e0413c9520f223b493c) a las diferencias con el master AVA (Si al hacer clic en el enlace, usted no puede encontrar el archivo `contributing.md` de los archivos modificados, se puede deducir que la traducción está actualizada).
___
```
1. Where the link `contributing.md` points to the file contributing.md from AVA
2. Where the link `enlace` points to the last commit respective translation (`65ae07c76b2a5927e6bcc00630a691c98f6e7c46`) followed by `...master#diff-` and sha file (`cc4aac3e9be04e0413c9520f223b493c`)

That is to do on the 3 files : contributing.md, maintaining.md and readme.md

File: es-ES/readme.md [hunk 2]
```diff
@@ -0,0 +1,703 @@
...
+Si devuelve un observable a partir de un test, AVA automáticamente podrá consumirlo hasta su finalización antes de terminar el test.
+
+*Si no necesita usar "callback mode" o llamar a `t.end()`.*
+
+```js
+test(t => {
+	t.plan(3);
+	return Observable.of(1, 2, 3, 4, 5, 6)
+		.filter(n => {
+			// only even numbers
```
Comment (forresst) [inline es-ES/readme.md]: This comment can be translated (// only even numbers)

File: es-ES/readme.md [hunk 3]
```diff
@@ -0,0 +1,703 @@
...
+$ npm test
+```
+
+
+## CLI
+
+```
+$ ava --help
+
+  Uso
```
Comment (forresst) [inline es-ES/readme.md]: I will not translate it, it is the output when taping `ava --help`

At best, translate into Spanish between two brackets:

```
Options
    --init       Add AVA to your project (Añadir AVA a tu proyecto)
```

File: es-ES/readme.md [hunk 4]
```diff
@@ -0,0 +1,703 @@
...
+test('bar', async t => {
+	t.plan(1);
+
+	const bar = Promise.resolve('bar').then(delay(200));
+
+	t.is(await bar, 'bar');
+});
+```
+
+<img src="screenshot.png" width="150" align="right">
```
Comment (forresst) [inline es-ES/readme.md]: To see correctly the image :
``
replace by : 
``

File: es-ES/readme.md [hunk 5]
```diff
@@ -0,0 +1,703 @@
...
+
+### Salida TAP opcional
+
+AVA puede generar una salida TAP a través de la opción `--tap` para su uso con cualquier [TAP reporter](https://github.com/sindresorhus/awesome-tap#reporters).
+
+```
+$ ava --tap | tap-nyan
+```
+
+<img src="media/tap-output.png" width="398">
```
Comment (forresst) [inline es-ES/readme.md]: To see correctly the image :
``
replace by : 
``

**Comment by forresst** [comment]
@AlbertoFuente @sindresorhus Except my remarks, it's beautiful work! Thank you

File: es-ES/readme.md [hunk 6]
```diff
@@ -0,0 +1,703 @@
...
+
+A pesar de que JavaScript se ejecuta en un solo hilo, IO en Node.js puede ejecutarse en paralelo debido a su naturaleza asíncrona. AVA se aprovecha de esto y corre sus tests al mismo tiempo, lo que es especialmente beneficioso para tests pesados en IO. Además, los archivos de test se ejecutan en paralelo como procesos separados, que le da un mejor rendimiento y un entorno aislado para cada archivo de test. [Cambiando](https://github.com/sindresorhus/pageres/commit/663be15acb3dd2eb0f71b1956ef28c2cd3fdeed0) de Mocha a AVA en Pageres llevó el tiempo de test por debajo de 31 segundos, concretamente a 11 segundos. El tener tests que se ejecutan al mismo tiempo nos obliga a escribir tests atómicos, es decir, los tests no dependen del estado global o el estado de otros tests, lo que está muy bien!
+
+*Lea nuestra [guía de contribución](contributing.md) si está pensando en contribuir (issues/PRs/etc).*
+
+
+## Tabla de contenidos
+
+- [Uso](#usage)
```
Comment (forresst) [inline es-ES/readme.md]: Bad link, replace `[Uso](#usage)` by `[Uso](#uso)`

File: es-ES/readme.md [hunk 7]
```diff
@@ -0,0 +1,703 @@
...
+- [Uso de CLI](#cli)
+- [Documentación](#documentation)
```
Comment (forresst) [inline es-ES/readme.md]: Bad link, replace `[Documentación](#documentation)` by `[Documentación](#documentacion)`

File: es-ES/readme.md [hunk 8]
```diff
@@ -0,0 +1,703 @@
...
+- [API](#api)
+- [Aserciones](#assertions)
```
Comment (forresst) [inline es-ES/readme.md]: Bad link, replace `[Aserciones](#assertions)` by `[Aserciones](#aserciones)`

File: es-ES/readme.md [hunk 9]
```diff
@@ -0,0 +1,703 @@
...
+
+
+## ¿Por qué AVA?
+
+- Mínimo y rápido
+- Sintaxis simple de test
+- Ejecuta tests concurrentemente
+- Fuerza a escribir tests atómicos
+- No globales implícitos
+- [Entorno aislado para cada archivo de test](#isolated-environment)
```
Comment (forresst) [inline es-ES/readme.md]: Bad link, replace `[Entorno aislado para cada archivo de test](#isolated-environment)` by `[Entorno aislado para cada archivo de test](#entorno-aislado)`

**It is necessary to check all the links**

**Comment by AlbertoFuente** [comment]
Thank you so much for your feedback @forresst , completely agree with everything, I will review the translations and links as soon as I can.

Keep in touch.

**Comment by AlbertoFuente** [comment]
@forresst - I have finished the translations and implement your improvements, can you please check that the format is correct, I have not clear if I've correctly done the part of `... master #diff -`.

Thanks

File: es-ES/readme.md [hunk 10]
```diff
@@ -0,0 +1,731 @@
...
+
+## FAQ
+
+### Por qué no `mocha`, `tape`, `node-tap`?
+
+Mocha requiere el uso de globales implícitas como `describe` y `it` por defecto (que utilizan la mayoría de las personas), sin demasiadas opiniones buenas, hinchado, síncrono por defecto, ejecuta los tests en serie y es lento. Tape y node-tap son bastante buenos. AVA está muy inspirado en su sintaxis. Sin embargo, ambos ejecutan los tests en serie y han hecho de [TAP](https://testanything.org) un ciudadano de primera clase que ha hecho en mi humilde opinión que sus códigos base sean un poco enrevesadas y acopladas. La salida de TAP es difícil de leer, así que siempre terminan usando una herramienta de reporte de TAP externa. AVA tiene bastantes buenas opiniones y es concurrente. Viene con un simple reporter por defecto y tiene voluntad en el futuro de dar apoyo a Tun TAP reporter.
+
+### ¿Cómo es el nombre escrito y pronunciado?
+
+AVA, no Ava o ava. Pornunciado [`/ˈeɪvə/` ay-və](media/pronunciation.m4a?raw=true).
```
Comment (forresst) [inline es-ES/readme.md]: Bad link: Replace

``` markdown
Pornunciado [`/ˈeɪvə/` ay-və](media/pronunciation.m4a?raw=true). 
```

BY

``` markdown
Pornunciado [`/ˈeɪvə/` ay-və](https://github.com/sindresorhus/ava/blob/master/media/pronunciation.m4a?raw=true).
```

File: es-ES/readme.md [hunk 11]
```diff
@@ -0,0 +1,731 @@
...
+
+### ¿Cuál es el fondo de la cabecera?
+
+[Andromeda galaxy.](https://simple.wikipedia.org/wiki/Andromeda_galaxy)
```
Comment (forresst) [inline es-ES/readme.md]: There is a link to Wikipedia in Spanish. it may be better? (`https://es.wikipedia.org/wiki/Galaxia_de_Andr%C3%B3meda`)

**Comment by forresst** [comment]
@AlbertoFuente Good job ! The part of `... master #diff -` is correct.
There's just three little things to do and it will be perfect for me, sorry, I had not seen it:
1. To keep the same naming of the French directory, replace the name directory `es-ES` by `es_ES`
2. Adds a row in the [readme.md](https://github.com/sindresorhus/ava-docs/blob/master/readme.md) to add the link to the Spanish language. (If possible before the french language to respect alphabetical order : 1: español, 2: français) : 
   
   ``` markdown
   - [Español](es_ES/readme.md) *(by [@AlbertoFuente](https://github.com/AlbertoFuente))*
   ```
3. Rebase your PR to one commit (It contains a [commit master](https://github.com/AlbertoFuente/ava-docs/commit/117ea92d2db2a16547620eec2e993cd850384aa6))

LGTM

@sindresorhus Other remarks or it's ok for you?

**Comment by AlbertoFuente** [comment]
@forresst - Already changed the name of the directory to 'es_ES', when we merge the changes I put the link to the Spanish language in the readme.md file.

**Comment by sindresorhus** [comment]
> when we merge the changes I put the link to the Spanish language in the readme.md file.

Just add it to this PR ;)

**Comment by AlbertoFuente** [comment]
@sindresorhus @forresst - Rebased PR to one commit, check that everything is correct :stuck_out_tongue_winking_eye:

**Comment by forresst** [comment]
> Adds a row in the [readme.md](https://github.com/sindresorhus/ava-docs/blob/master/readme.md) to add the link to the Spanish language. (If possible before the french language to respect alphabetical order : 1: español, 2: français) : 
> 
> ``` markdown
>    - [Español](es_ES/readme.md) *(by [@AlbertoFuente](https://github.com/AlbertoFuente))*
> ```

I think a little forgetfulness : respect alphabetical order
Thank you again for your work.
LGTM

**Comment by sindresorhus** [comment]
> I think a little forgetfulness : respect alphabetical order

Fixed it on merge.

**Comment by sindresorhus** [comment]
Thank you @AlbertoFuente! :)

**Comment by sindresorhus** [comment]
Added you to the repo. Feel free to commit updates directly.

**Comment by AlbertoFuente** [comment]
Thank you @sindresorhus and @forresst also, we are in contact for future updates

**Pull request closed without merging**